winml/adapter/WinMLAdapter.cpp

```diff
@@ -638,14 +638,21 @@ class WinMLAdapter : public Microsoft::WRL::RuntimeClass<
       onnxruntime::IExecutionProvider* provider,
       OrtAllocator** allocator) override try {
     auto allocator_ptr = provider->GetAllocator(0, ::OrtMemType::OrtMemTypeDefault);
-    *allocator = new AllocatorWrapper(allocator_ptr);
+    *allocator = new (std::nothrow) AllocatorWrapper(allocator_ptr);
```

Comment:
This is unrelated to this fix, but I assume it must be nothrow since the next line compares with nullptr

Reply:
nice !

---
In reply to: [356904692](https://github.com/microsoft/onnxruntime/pull/2635#discussion_r356904692) [](ancestors = 356904692)